static/js/jira-select-card-option.js

```diff
@@ -7,7 +7,16 @@ function openChildWindow(url) {
 	const interval = setInterval(function () {
 		if (child.closed) {
 			clearInterval(interval);
```

Comment:
Is this just checking if the child page closes and then navigating back to the config page? This would work for the scenario where users click 'Connect' (cloud or server path) as at this point we're expecting them to go back to the config page. But have you thought about what will happen when a user clicks the back narrow in the navigation from the /github/configuration page?

2 pages import this js file that would be impacted by this implementation: github version page and the auto/manual screen. 

- If I go down the cloud install path, I'll go from the GH version screen to  /github/configuration but if I click the back button, I expect to end up back on the GH version page. 
- Likewise, if I go down the server install path and choose to automatically create my app, the last page I'll be on before I get to  /github/configuration is the auto/manual app view. I would expect to return to the manual/auto app page, not the GH config page.